src/ocean/core/Option.d

```diff
@@ -0,0 +1,236 @@
+/******************************************************************************
+
+    Wraps any type in a struct that also contains boolean field indicating if
+    value is in defined state.
+
+    If T is a value type, `Option!(T)` is value type too.
+
+    Copyright:
+        Copyright (c) 2009-2016 Sociomantic Labs GmbH.
+        All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE_BOOST.txt for details.
+        Alternatively, this file may be distributed under the terms of the Tango
+        3-Clause BSD License (see LICENSE_BSD.txt for details).
+
+
+*******************************************************************************/
+
+module ocean.core.Option;
+
+version(UnitTest)
```

Comment:
Style: Lack of spacing before parenthesis everywhere

Reply:
It is not required by style guide. Ocean has both styles with more recent code leaning towards lack of space (though I am probably to blame for this, being author of some added packages).